doc/create-a-custom-rule.md

```diff
@@ -0,0 +1,211 @@
+# Create a custom `remark-lint` rule
+
+This tutorial will help you setting up a JavaScript project, and creating your first linting plugin for `remark`.
+
+### Table of Contents
+
+*   [Set up the project](#set-up-the-project)
+*   [Set up remark](#set-up-remark)
+*   [The no-invalid-gif rule:](#the-no-invalid-gif-rule)
+*   [Create the custom rule](#create-the-custom-rule)
+*   [Rule arguments](#rule-arguments)
+*   [Rule implementation](#rule-implementation)
+*   [Import the rule in your remark config](#import-the-rule-in-your-remark-config)
+*   [Apply the rule](#apply-the-rule)
+
+\## Set up the project
```

Comment:
@wooorm 
not sure why I get this autofix for this line:

```bash

doc/create-a-custom-rule.md
  7:5-7:46  warning  Link to unknown heading: `set-up-the-project`  missing-heading  remark-validate-links
```

it only happens for this first header.